### ISSUE TITLE
unable to directly add the transaction address into the address book 

### ISSUE THREAD

**Issue by oddiccey001** [NewEconomyMovement/NEMiOSApp#178]
attached the video
[address book.zip](https://github.com/NewEconomyMovement/NEMiOSApp/files/201468/address.book.zip)

To make it more user friendly it would be nice if you can have a button on the transactions screen that would state "add to address book" and would directly bring up the address book, and you only would need to add the name as the address would automatically upload. 

Also the iphone address book is directly linked to the app, so if you delete one of your contacts from within the app it deletes from your iphone, which is not ideal.

**Comment by jabo38** [comment]
yes. we will be taken a look at these issues in the next major upgrade of this app after we launch on mainnet, but this version won't get those upgrades yet.
